IronSoftware.Drawing/IronSoftware.Drawing.Common/IronSoftware.Drawing.Common.csproj

```diff
@@ -12,7 +12,7 @@
 		<NoWarn>CS8002</NoWarn>
 		<Platforms>AnyCPU</Platforms>
 		<SignAssembly>true</SignAssembly>
-		<TargetFrameworks>netstandard2.0;netstandard2.1;net60</TargetFrameworks>
+		<TargetFrameworks>netstandard2.0;net60</TargetFrameworks>
```

Comment:
I'm just curious. Why is `netstandard2.1` removed?

Reply:
`netstandard2.1` similar to `netstandard2.0` you can check out [netstandard2.0](https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-0) and [netstandard2.1](https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-1)